libs/@local/hash-graph-sdk/node/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@local/hash-graph-sdk",
+  "version": "0.0.0-private",
+  "private": true,
+  "description": "SDK for the HASH Graph API.",
```

Comment:
It's not really for the API, right? it doesn't communicate with it... a Graph API SDK would be some combination of HaRPC and this. Maybe just 'SDK for the HASH Graph' for now?

Reply:
```suggestion
  "name": "@local/hash-graph-sdk",
  "version": "0.0.0-private",
  "private": true,
  "description": "SDK for the HASH Graph.",
```